### DISCUSSION TITLE
部署失败

### DISCUSSION THREAD

**Discussion by Heffie199** [secretflow/secretflow#792] [Q&A]
[root@host-15-0-0-236 ap]# tar -xf secretflow-allinone-package-latest.tar.gz
tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.md5ex'
[root@host-15-0-0-236 ap]# cd /ap/secretflow-allinone-package/
[root@host-15-0-0-236 secretflow-allinone-package]# ll
total 8
drwxr-xr-x 2 501 games 4096 Jul 27 13:53 images
-rwxr-xr-x 1 501 games 1399 Jul  7 11:38 install.sh
[root@host-15-0-0-236 secretflow-allinone-package]# ./install.sh
images/kuscia-0.2.0b1.tar
echo Loaded image: secretflow/kuscia:0.2.0b1
images/secretflow-lite-anolis8-1.0.0b2.tar
echo Loaded image: secretflow-registry.cn-hangzhou.cr.aliyuncs.com/secretflow/secretflow-lite-anolis8:1.0.0b2
images/secretpad:0.1.0b0.tar
echo Loaded image: secretflow-registry.cn-hangzhou.cr.aliyuncs.com/secretflow/secretpad:0.1.0b0
WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
exec /home/kuscia/bin/tini: exec format error
什么问题啊 都docker安装好了

**Comment by Mingbo-Lee** [comment]
看起来是平台不匹配的问题
```
WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
```
您的宿主机是什么类型的CPU？
